app/decorators/manageiq/providers/container_manager_decorator.rb

```diff
@@ -0,0 +1,18 @@
+module ManageIQ::Providers
+  class ContainerManagerDecorator < MiqDecorator
+    def quadicon(_n = nil)
+      {
+        :top_left     => {:text => container_nodes.size},
+        :top_right    => {:state_icon => enabled? ? "on" : "paused"},
```

Comment:
where is the on/paused info taken from?

Reply:
https://github.com/ManageIQ/manageiq-ui-classic/pull/3046#discussion_r156927784 it's from refactored code (quadicon_helper.rb:571)[https://github.com/ManageIQ/manageiq-ui-classic/pull/3046/files#diff-6525cb9600835fcc2ba062fde39a2292L571]